db/migrate/20211129112149_add_role_to_user.rb

```diff
@@ -1,5 +1,5 @@
 class AddRoleToUser < ActiveRecord::Migration[6.1]
   def change
-    add_column :users, :role_id, :integer
+    add_references :users, :role_id, :integer
```

Comment:
this should bring changes to `db/schema.rb` file. Let's try `rails db:rollback && rails db:migrate` and check in any changes.